src/components/Chip/Chip.tsx

```diff
@@ -49,22 +51,28 @@ class Chip extends React.PureComponent<Props, any> {
         : ''
     ),
       <span key="2">
-        {children}
-      </span>,
+      {children}
+    </span>,
     ];
     const isClickable = clickable ?
-      <a onClick={onClick} aria-disabled={false} tabIndex={0}>
+      <a onClick={onClick} aria-disabled={false}>
         {chipContents}
       </a>
       : chipContents;
     const isRemovable = removable ?
-      <button type="button" className={theme.Remove} aria-label={'Remove ' + children} onClick={onRemove}>
+      <a className={theme.Remove} aria-label={'Remove ' + children} onClick={onRemove} tabIndex={-1}>
         <span aria-hidden="true">×</span>
-      </button>
+      </a>
       : '';
-    const returnedValue = clickable || removable ? React.createElement('span', { className }, isClickable, isRemovable) : React.createElement('button', { className }, isClickable, isRemovable);
+    const returnedValue = clickable || removable ? React.createElement('button', { className }, isClickable, isRemovable) : React.createElement('button', { className }, isClickable, isRemovable);
```

Comment:
Both conditions are now same. So, remove returned value variable and use single button react element creation.

Reply:
resolved